lib/rubocop/cop/mixin/alignment.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # This module does auto-correction of nodes that should just be moved to
```

Comment:
I guess you should update the description of the mixins, as now they are not doing auto-correction anymore.

Reply:
I cleaned up this description and went through all the other mixins I adjusted and checked that none mentioned autocorrection.